atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/AllowedRangeRequest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.api;
+
+/**
+ * Mark that this range request has been thought about and explicitly decided to be ok.
+ */
+public @interface AllowedRangeRequest {
```

Comment:
I was wondering if having an enum field for high level categorisation would be helpful for future analysis for why/for what services are relying on rangerequests. Like background task, search use case, backfill, etc?